ui/widgets/nuxeo-selectivity.js

```diff
@@ -7299,6 +7310,14 @@ typedArrayTags[weakMapTag] = false;
 
     _placeholderChanged() {
       this.$.input.setAttribute('placeholder', this.placeholder);
+      const multipleInput = this.shadowRoot.querySelector('.selectivity-multiple-input');
+      if (this.multiple && multipleInput) {
+        multipleInput.setAttribute('placeholder', this.placeholder);
+      }
+      const singleInputPlaceholder = this.shadowRoot.querySelector('.selectivity-placeholder');
```

Comment:
This part should be placed in an else statement, right? There's no point in running this if it is multiple.